internal/services/cognitive/ai_services_resource.go

```diff
@@ -427,6 +417,18 @@ func (AzureAIServicesResource) Read() sdk.ResourceFunc {
 					if props.DisableLocalAuth != nil {
 						localAuthEnabled = !*props.DisableLocalAuth
 					}
+
+					if localAuthEnabled {
+						keys, err := client.AccountsListKeys(ctx, *id)
+						if err != nil {
+							return fmt.Errorf("listing the Keys for %s: %+v", id, err)
+						}
+
+						if model := keys.Model; model != nil {
+							state.PrimaryAccessKey = pointer.From(model.Key1)
+							state.SecondaryAccessKey = pointer.From(model.Key2)
+						}
+					}
```

Comment:
Can you please update the documentation with a note below these properties with the explanation that these are only available if `location_authentication_enabled` is `true`